codegenerator/cli/npm/envio/src/ReorgDetection.res

```diff
@@ -186,39 +215,59 @@ module LastBlockScannedHashes: {
     ~currentBlockHeight,
   ) => {
     let verifiedDataByBlockNumber = Js.Dict.empty()
-    blockNumbersAndHashes->Array.forEach(blockData => {
```

Comment:
Surely Array.forEach is faster?

Reply:
for loop is faster. But I needed the change to use ref inside, I didn't want it to be compiled as an object. But then I changed my mind.